installer/functions/common.ts

```diff
@@ -0,0 +1,58 @@
+import { Context, TwilioClient } from "@twilio-labs/serverless-runtime-types/types";
+import { ApplicationContext } from "../src/constants/interface";
+
+// constants
+export const IS_LOCALHOST = "IS_LOCALHOST";
+export const ACCOUNT_SID = "ACCOUNT_SID";
+export const AUTH_TOKEN = "AUTH_TOKEN";
+export const LOCAL_HOST = "localhost:";
+
+export function getParam(context: Context, key: string) {
```

Comment:
what convenience does this provide you over calling `isLocalhost`/`getAccountSid`/`getAuthToken` directly?

Reply:
for example, i think if linting was implemented it should've reported this function. `getAccountSid`/`getAuthToken` return a Promise<string> while `isLocalhost` returns a synchronous type